internal/ui/logger_ui.go

```diff
@@ -1,23 +1,18 @@
 package ui
 
 import (
-	"io"
-
 	"github.com/anchore/syft/internal/log"
 	syftEvent "github.com/anchore/syft/syft/event"
 	"github.com/wagoodman/go-partybus"
 )
 
 type loggerUI struct {
-	unsubscribe  func() error
-	reportOutput io.Writer
+	unsubscribe func() error
 }
 
 // NewLoggerUI writes all events to the common application logger and writes the final report to the given writer.
```

Comment:
I didn't quite follow how the events are written to the "common application logger" here?

Reply:
yeah, the phrasing is a bit weird. We could probably even rename this to "NopUI" since it doesn't respond to any events and the caller is expected to have configured the logger to write out to stderr (and be the UI).